### PR TITLE
docs(mikroorm): add default mikro-orm howto

### PR DESCRIPTION
**Pull request by boredland**
## Changes 💡

contributes to https://github.com/nextauthjs/adapters/pull/325

## Screenshot (If Applicable) 📷

image

### CODE REVIEW

File: docs/adapters/mikro-orm.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+```
+
+Configure NextAuth.js to use the MikroORM Adapter:
+
+```typescript title="pages/api/auth/[...nextauth].ts"
+import NextAuth from "next-auth"
+import GoogleProvider from "next-auth/providers/google"
+import { MikroOrmAdapter } from "@next-auth/mikro-orm-adapter"
+
+const prisma = new PrismaClient()
```
Comment (balazsorban44) [inline docs/adapters/mikro-orm.md:27]:
```suggestion
```
Leftover? :smile:

File: docs/adapters/mikro-orm.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+export default NextAuth({
+  adapter: MikroOrmAdapter({
+    // MikroORM options object. Ref: https://mikro-orm.io/docs/next/configuration#driver
+    dbName: "./db.sqlite",
+    type: "sqlite",
+    // you can add e
+    entities: [],
```
Comment (balazsorban44) [inline docs/adapters/mikro-orm.md:35]:
```suggestion
```

Just skip by default, so people don't copy it unnecessarily.

File: docs/adapters/mikro-orm.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+
+```bash npm2yarn
+npm install next-auth@beta @next-auth/mikro-orm-adapter@next @mikro-orm/core@next @mikro-orm/[YOUR DRIVER]@next
+```
+
+Configure NextAuth.js to use the MikroORM Adapter:
+
+```typescript title="pages/api/auth/[...nextauth].ts"
+import NextAuth from "next-auth"
+import GoogleProvider from "next-auth/providers/google"
```
Comment (balazsorban44) [inline docs/adapters/mikro-orm.md:24]:
```suggestion
```

File: docs/adapters/mikro-orm.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+    // you can add e
+    entities: [],
+    debug: process.env.DEBUG === "true" || process.env.DEBUG?.includes("db"),
+  }),
+  providers: [
+    GoogleProvider({
+      clientId: process.env.GOOGLE_CLIENT_ID,
+      clientSecret: process.env.GOOGLE_CLIENT_SECRET,
+    }),
+  ],
```
Comment (balazsorban44) [inline docs/adapters/mikro-orm.md:43]:
```suggestion
  providers: [],
```

Shaves off a few lines.

**Review by balazsorban44 — CHANGES_REQUESTED** [review]

File: docs/adapters/mikro-orm.md (hunk 1 above)
Comment (boredland) [reply docs/adapters/mikro-orm.md:27]: 🗡️ I am the king 👑 of copy and pasta 🍝

File: docs/adapters/mikro-orm.md [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+:::warning
+Please make sure to use the `next` tagged version of mikro-orm. Refer [here](https://github.com/jonahallibone/mikro-orm-nextjs) for a setup example with NextJS.
+:::
+
+:::warning
+Please make sure to use the `next` tagged version of the adapter.
+:::
+
+```bash npm2yarn
+npm install next-auth@beta @next-auth/mikro-orm-adapter@next @mikro-orm/core@next @mikro-orm/[YOUR DRIVER]@next
```
Comment (balazsorban44) [inline docs/adapters/mikro-orm.md:17]:
```suggestion
npm install next-auth @next-auth/mikro-orm-adapter @mikro-orm/core@next @mikro-orm/[YOUR DRIVER]@next
```

Actually, just drop the `next`, `beta`! Next stop is at `latest` :steam_locomotive:

**Review by boredland — COMMENTED** [review]
.

File: docs/adapters/mikro-orm.md [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+
+To use this Adapter, you need to install Mikro ORM, the driver that suits your database, and the separate `@next-auth/mikro-orm-adapter` package:
+
+:::warning
+Please make sure to use the `next` tagged version of mikro-orm. Refer [here](https://github.com/jonahallibone/mikro-orm-nextjs) for a setup example with NextJS.
+:::
+
+:::warning
+Please make sure to use the `next` tagged version of the adapter.
+:::
```
Comment (balazsorban44) [inline docs/adapters/mikro-orm.md:14]:
```suggestion
```

**Review by balazsorban44 — APPROVED** [review]

**Pull request merged**